doc/netplan-yaml.md

```diff
@@ -12,6 +12,7 @@ network:
   renderer: STRING
   bonds: MAPPING
   bridges: MAPPING
+  dummies: MAPPING
```

Comment:
Hmm, this is difficult.  While `dummy` is clearly correct, the plural `dummies` evokes for me images of mannequins or crash test dummies or martial arts training dummies.  At the risk of being too long, what do you think of `dummy-interfaces` here?

Reply:
oh `dummy-interfaces` would deviate from the naming pattern... `ethernets`, `bonds`, `bridges`... `dummy-interfaces`... as a non-native English speaker I wasn't seeing the type of dummies you mentioned but now I can't unsee them :D

If we are not going with `dummies`, maybe we could go with `dummy-devices` then? Because we already have a `nm-devices` category.

Reply:
works for me

Reply:
Same for me... I didn't think about cash test dummies and thought "dummies" would be most in line with what we have so far for the other types. But I see Steve's point. Considering we also have "nm-devices", I think I'd go with "dummy-devices" instead of "dummy-interfaces".